wls-gui-wahllokalsystem/tests/types/vorfaelleundvorkommnisse/Ereignisart.spec.ts

```diff
@@ -1,13 +1,33 @@
-import { describe, expect, it } from "vitest";
+import { createTestingPinia } from "@pinia/testing";
+import { beforeEach, describe, expect, it, vi } from "vitest";
 
+import { useUserStore } from "@/stores/userStore.ts";
+import { User } from "@/types/User.ts";
 import {
   EreignisartEnum,
   getEreignisArtForDateRelatedToSchliessungsuhrzeit,
 } from "@/types/vorfaelleundvorkommnisse/Ereignisart.ts";
+import { WahlbezirksArtEnum } from "@/types/wahlbezirksArtEnum.ts";
 
 describe("Ereignisart.ts", () => {
-  describe("getEreignisArtForDateRelatedToSchliessungsuhrzeit", () => {
-    it("should_returnVorfall_when_schliessungsuhrzeitIsNotSet", () => {
+  let userStore: ReturnType<typeof useUserStore>;
+
+  beforeEach(() => {
+    const testPinia = createTestingPinia({
+      stubActions: false,
+      createSpy: vi.fn,
+    });
+    userStore = useUserStore(testPinia);
+  });
+
+  describe("getEreignisArtForDateRelatedToSchliessungsuhrzeitForUWB", () => {
+    beforeEach(() => {
+      const user = new User();
+      user.wahlbezirksArt = WahlbezirksArtEnum.UWB;
+      userStore.setUser(user);
```

Comment:
bitte überall den neuen builder aus der UserTestDataFactory nutzen
```suggestion
      userStore.setUser(prepareUser().wahlbezirksArt(WahlbezirksArtEnum.UWB).build());
```